userland/examples/rf233/rf233.c

```diff
@@ -183,9 +183,9 @@ void trx_bit_write(uint8_t reg_addr,
 void trx_sram_read(uint8_t addr, uint8_t *data, uint8_t length)  {
   spi_hold_low();
   /* Send the command byte */
-  uint8_t tmp1 = spi_write_byte(TRX_CMD_SR);
+  uint8_t __attribute__ ((unused)) tmp1 = spi_write_byte(TRX_CMD_SR);
   /* Send the command byte */
-  uint8_t tmp2 = spi_write_byte(addr);
+  uint8_t __attribute__ ((unused)) tmp2 = spi_write_byte(addr);
```

Comment:
I don't love this, because in the not `_DEBUG_` case, it marks things as `unused` that are used, which is weird, but I also don't have a better solution. The usual approach I've seen is the have a function called `debug`, where the function body is conditional, something like

```c
void debug(const char* str, ...) {
#ifdef DEBUG
  va_list v;
  va_start(v, str);
  vprintf(str, v);
  va_end(v);
#else
  (void)sizeof(str);
#endif
}
```